src/test/java/Homework22.java

```diff
@@ -0,0 +1,27 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework22 extends BaseTest {
+
+    @Test
+    public void renamePlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Reply:
@Berichicko I deleted the exception part but can you please explain why it is not needed here?

Reply:
> @Berichicko I deleted the exception part but can you please explain why it is not needed here?

The InterruptedException is declared in the method signature (throws InterruptedException), but it's not needed because the method doesn't actually throw this exception. It's unnecessary and can be removed for cleaner and simpler code.